### PR TITLE
feat: add MediaMetadata

### PR DESCRIPTION
**Pull request by Hashen110**
Screenshot from 2022-01-27 16-47-40

add [MediaMetadata](https://developer.mozilla.org/en-US/docs/Web/API/MediaMetadata)

### CODE REVIEW

**Comment by Hashen110** [comment]
> Screenshot from 2022-01-27 16-47-40

[MediaMetadata
](https://developer.mozilla.org/en-US/docs/Web/API/MediaMetadata)

**Comment by Hashen110** [comment]
[MediaMetadata](https://developer.mozilla.org/en-US/docs/Web/API/MediaMetadata)

**Comment by Hashen110** [comment]
[Customize media notifications and playback controls outside the browser](https://www.youtube.com/watch?v=Ft6diPReUAU&ab_channel=GoogleChromeDevelopers)

File: package.json [hunk 1]
```diff
@@ -28,7 +28,10 @@
   "bugs": {
     "url": "https://github.com/sampotts/plyr/issues"
   },
-  "browserslist": "> 1%",
+  "browserslist": [
+    "cover 100%",
+    "not dead"
+  ],
```
Comment (sampotts) [inline package.json:34]: What's the reason behind this change?

File: src/js/controls.js [hunk 2]
```diff
@@ -1745,6 +1745,21 @@ const controls = {
...
+      if ('mediaSession' in navigator) {
+        navigator.mediaSession.metadata = new window.MediaMetadata({
+          title: this.config.mediaMetadata.title,
+          artist: this.config.mediaMetadata.artist,
+          album: this.config.mediaMetadata.album,
+          artwork: this.config.mediaMetadata.artwork,
+        });
+      }
+      // eslint-disable-next-line no-empty
+    } catch (e) {}
```
Comment (sampotts) [inline src/js/controls.js:1761]: Usually, I just do:
```javascript
catch(e) {
  // Do nothing
}
```

**Review by sampotts — COMMENTED** [review]
Thanks! I just had one question re: the browserslist change but that's it. The other comment is just a nitpick and can be ignored if you want.

File: package.json (hunk 1 above)
Comment (Hashen110) [reply package.json:34]: My IDE (webstorm) showed a warning that string is not a valid option for the browserslist.

Comment (Hashen110) [reply package.json:34]: And the current value for browserslist does not do anything, I guess

Comment (Hashen110) [reply package.json:34]: Before changing the browserslist

After changing the browserslist

Comment (sampotts) [reply package.json:34]: Sorry, your IDE has a bug I guess. It's definitely [valid](https://browserslist.dev/?q=PiAxJQ%3D%3D) and is widely used. The CSS property [`transition`](https://caniuse.com/css-transitions) and [`align-items`](https://caniuse.com/mdn-css_properties_align-items_flex_context) no longer need prefixing so your first screenshot (before the change) is correct.

**Review by sampotts — COMMENTED** [review]
Thanks! I've reverted the `browserslist` change as discussed.

**Review by sampotts — APPROVED** [review]

**Comment by sampotts** [comment]
Thanks for the work on this. I've made a few small changes to the API and UI for it but it's mostly as-is! Thanks again. It's in v3.7.0 🎉

**Comment by sampotts** [comment]
Thanks for the work on this. It's included in v3.7.0 🎉
